boards/same54-xpro/include/board.h

```diff
@@ -75,6 +75,14 @@ extern "C" {
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name ATA6561 STANDBY pin definition
+ * @{
+ */
+#define AT6561_STBY_PIN            GPIO_PIN(PC, 13)
+#define AT6561_STBY_MODE           GPIO_OUT
```

Comment:
add exceptions to `https://github.com/RIOT-OS/RIOT/tree/master/dist/tools/doccheck`
it think an `exclude_simple` pattern should fit